README.rst

```diff
@@ -110,11 +112,12 @@ __ `FIWARE Policy Manager GE: Facts`_
 Configuration file
 ------------------
 The configuration used by the fiware-facts component is optionally read from the file
-``conf/fiware-facts.cfg``
+``facts_conf/fiware-facts.cfg``
 
 Default values are found in ``facts/config.py``.
```

Comment:
Have we got two configuration files? I can see that each file has a different representation for managing properties.